IETF-Hackathon/pqc-certificates
# general OQS update

General update to "everything OQS":
- Removal of support for EOL (oqs-)openssl111
- Generating new artifacts and storing them in ZIP again: (oqsprovider)/artifacts.zip
- moved old "artifacts.zip" to suitably named files should anyone want to test against those
- Improved certificate generation to include critical R3 extensions
- Added logic to generate CSV "compatMatrices" files
- Added "cross_verify" results
- Added logic to generate OCSP certificates (and test it via an OCSP responder)

Open issues:
- How to test (and report) OCSP results?
- Collating the compatMatrices results into [overall results file](https://github.com/IETF-Hackathon/pqc-certificates/blob/master/docs/pqc_hackathon_results.md): Is there a script for this @johngray-dev ?
- Testing other implementations with these new artifacts: I wasn't able to run other "gen.sh" scripts (see https://github.com/IETF-Hackathon/pqc-certificates/issues/63, https://github.com/IETF-Hackathon/pqc-certificates/issues/64) -- but didn't go to the length of building implementations from scratch to try this. Thanks @CBonnell and @ueno for making available docker images.
- Evaluate results more deeply: It seems interop with digicert and BC is pretty OK; everything else seems to be bad.
